src/plots/gl3d/scene.js

```diff
@@ -260,21 +262,45 @@ function initializeGLPlot(scene, pixelRatio, canvas, gl) {
 
     var gd = scene.graphDiv;
 
+    var makeUpdate = function() {
+        var update = {};
+
+        // camera updates
+        update[scene.id + '.camera'] = getLayoutCamera(scene.camera);
+
+        // scene updates
+        update[scene.id + '.aspectratio'] = scene.glplot.getAspectratio();
```

Comment:
This should **only** part of the event data on scene with orthographic projections

Reply:
... in other words, the event data should have the minimal set of keys that changed during the (here: scroll) interaction.

Reply:
Done in https://github.com/plotly/plotly.js/pull/4292/commits/b92230a0b99f883dd009e89890afc7555450a7a2.

Reply:
And in https://github.com/plotly/plotly.js/pull/4292/commits/d825c0e18de420cb15cd30747f366e394729bff7.